From 1e6cff0b3b5c1d1ceffd5ddc3124f3bb8e2e78da Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Thu, 21 Jan 2010 01:10:28 +0100
Subject: [PATCH] keymap: Fix invalid map line

Fix invalid line in keymaps/lenovo-thinkpad-usb-keyboard-trackpoint which did
not assign any key to a scan code.
---
 .../lenovo-thinkpad-usb-keyboard-trackpoint        |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/extras/keymap/keymaps/lenovo-thinkpad-usb-keyboard-trackpoint b/extras/keymap/keymaps/lenovo-thinkpad-usb-keyboard-trackpoint
index 2566f03..7612f79 100644
--- a/extras/keymap/keymaps/lenovo-thinkpad-usb-keyboard-trackpoint
+++ b/extras/keymap/keymaps/lenovo-thinkpad-usb-keyboard-trackpoint
@@ -10,4 +10,4 @@
 0x90011 prog1 # Thinkvantage button
 
 0x90015 camera # Fn+F6 headset/camera VoIP key  ??
-0x90010 # Microphone mute button  ??
+# 0x90010 # Microphone mute button  ??
-- 
1.7.3.2

